<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XingApiTest</class>
 <widget class="QWidget" name="XingApiTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XingApiTest</string>
  </property>
  <widget class="QListWidget" name="logViewWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dumpButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>112</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>dump</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>53</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
